pom.xml

```diff
@@ -92,6 +92,12 @@
 			<artifactId>commons-lang3</artifactId>
 			<version>${commons.lang3.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.12</version>
+			<scope>test</scope>
+		</dependency>
```

Comment:
Speculating as I normally do but as they didn't have tests or JUnit previously then are you sure these tests run successfully outside the IDE?

Reply:
Done